experiments/AMIP/coupler_driver.jl

```diff
@@ -152,11 +152,7 @@ the plots (from postprocessing and the conservation checks) of the simulation wi
 temporary files will be saved.
 =#
 
-if isinteractive()
-    COUPLER_OUTPUT_DIR = joinpath("output", joinpath(mode_name, run_name)) # TempestRemap fails if interactive and paths are too long
-else
-    COUPLER_OUTPUT_DIR = joinpath(pkg_dir, "experiments/AMIP/output", joinpath(mode_name, run_name))
-end
+COUPLER_OUTPUT_DIR = joinpath(config_dict["coupler_output_dir"], joinpath(mode_name, run_name)) # TempestRemap fails if interactive and paths are too long
```

Comment:
Should we change "coupler_output_dir" in the interactive_debug.yml config file so we still have a shorter path name for interactive runs?

Reply:
Sure! That's a great idea! 

And the README is out of date, so I've removed it. 

Thank you!